.circleci/config.yml

```diff
@@ -49,7 +49,7 @@ jobs:
     steps:
       - checkout
       - ruby/install-deps
-      - run: bundle exec brakeman -q --no-pager -i brakeman.ignore
+      - run: bundle exec brakeman -q --no-pager --report-direct -i brakeman.ignore
```

Comment:
i think it might be better to add something to the brakeman.ignore file instead of doing a global flag. we have precedence for this.

i think it's bundle exec brakeman -i and then point it to the existing file when prompted

Reply:
Done.